thewalrus/quantum.py

```diff
@@ -659,6 +659,70 @@ def gen_Qmat_from_graph(A, n_mean):
     return Q
 
 
+def photon_number_var(mu, cov, j, k, hbar=2):
+    r""" Calculate the variance/covariance of the photon number distribution
+    of a Gaussian state.
+
+    Last two eq. of Part II. in 'Multidimensional Hermite polynomials and
+    photon distribution for polymode mixed light', Dodonov et al.
+    (https://journals.aps.org/pra/abstract/10.1103/PhysRevA.50.813).

```

Comment:
Would be nice to format the journal title as a link, i.e., 

```suggestion
    Last two eq. of Part II. in `'Multidimensional Hermite polynomials and
    photon distribution for polymode mixed light', Dodonov et al.
    <https://journals.aps.org/pra/abstract/10.1103/PhysRevA.50.813>`_.
```

(not 100% sure if a multiline ReST link will format correctly in the docs!)